tests/README.md

```diff
@@ -10,6 +10,9 @@ python3.7 -m venv venv/
 # install requirements
 venv/bin/pip install -r requirements.txt
 
+# generate sql
+venv/bin/python script/generate_sql
```

Comment:
It shouldn't be strictly necessary to execute the SQL generation manually. [conftest.py](https://github.com/mozilla/bigquery-etl/blob/master/tests/conftest.py) configures PyTest so that it should generate the SQL every time before executing the tests.

Reply:
I see that now. It wasn't running correctly for me earlier, and I have to generate the sql files explicitly. I'll investigate further